maas-schemas/schemas/core/components/spaceDemand.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$id": "http://maasglobal.com/core/components/spaceDemand.json",
+  "description": "A spaceDemand describes nr of passengers (adults, infatnts, etc) and bags (luggages).",
+  "definitions": {
+    "spaceDemandObject": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "adults": { "type": "integer", "minimum": 0, "description": "Number of adults" },
+        "children": { "type": "integer", "minimum": 0, "description": "Number of children" },
+        "infants": { "type": "integer", "minimum": 0, "description": "Number of infants" },
+        "bags": { "type": "integer", "minimum": 0, "description": "Number of bags (luggages)" }
```

Comment:
If you're doing the changes we could extend this with `pets` as well or it can be added later when needed. I'd say some taxi companies actually support animals like dogs.